redirects.go

```diff
@@ -118,13 +114,13 @@ func ParseString(s string) ([]Rule, error) {
 	return Parse(strings.NewReader(s))
 }
 
-// parseStatus returns the status code and force when "!" suffix is present.
-func parseStatus(s string) (code int, force bool, err error) {
+// parseStatus returns the status code.
+func parseStatus(s string) (code int, err error) {
 	if strings.HasSuffix(s, "!") {
-		force = true
-		s = strings.Replace(s, "!", "", -1)
+		// See https://docs.netlify.com/routing/redirects/rewrites-proxies/#shadowing
+		return -1, fmt.Errorf("forced redirects (or \"shadowing\") are not supported by IPFS gateways")
```

Comment:
I avoided "aka" and thought "also known as" might be long for an error message.  Including links seems bad and prone to needing updating.  I think I personally would prefer just "forced redirects" but if "shadowing" helps people, great. 👍